pkg/histogram/sparse_histogram.go

```diff
@@ -65,6 +65,14 @@ func (s SparseHistogram) Copy() SparseHistogram {
 	return c
 }
 
+func CountSpans(spans []Span) int {
```

Comment:
That's more "count buckets" or "count buckets in spans", right?

Reply:
Not important right now, but now that it is in my head: Perhaps declare a `type Spans []Span` and give it a method `func (s Spans) TotalLen() int`...

Reply:
makes sense